code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -216,6 +226,17 @@ GLOBAL_LIST_INIT(prglass_recipes, list (
 	point_value = 23
 	table_type = /obj/structure/table/glass/reinforced/plasma
 
+/obj/item/stack/sheet/plasmarglass/examine_more(mob/user)
+	. = ..()
+	. = list()
```

Comment:
what's the point of assigned the return value to `.` if you're going to immediately override it with `. = list()`?

Reply:
The proc **requires** that you call parent or it will kill you.

The empty list is not strictly required in this instance, but is used in cases where a parent and child object have different descriptions - the empty list stops the child from returning the parent description in addition to its own.

Reply:
Why not just do
```suggestion
	..()
	. = list()
```
Then

Also, unless someone overrides `examine_more()` lower down the type tree (which they shouldn't do), it's better to keep it like this, because then this issue is more apparent

Imo, this is the best solution
```suggestion
	. = ..()
```